### PR TITLE
DHC-1062 rebooting rug server (machine) screws up the management interface

### PR DESCRIPTION
**Pull request by rodis**

### CODE REVIEW

**Comment by rodis** [comment]
This is a work in progress because I didn't add the tests for the new method yet.
BTW the implementation should be complete so feel free to give a look and add comments.

File: akanda/rug/main.py [hunk 1]
```diff
@@ -21,6 +22,9 @@ def shuffle_notifications(notification_queue, sched):
             sched.handle_message(target, message)
         except KeyboardInterrupt:
             sched.stop()
+            # Purge the mgt tap interface on shutdown
```
Comment (dhellmann) [inline akanda/rug/main.py]: This should happen in main() since that's where we set up the interface in the first place.

File: akanda/rug/main.py [hunk 2]
```diff
@@ -29,7 +30,33 @@ def main(argv=sys.argv[1:]):
         cfg.StrOpt('host',
                    default=socket.getfqdn(),
                    help="The hostname Akanda is running on"),
+
+        # Note(rods): This options, togheter with the AGENT_OPTIONS
```
Comment (markmcclain) [inline akanda/rug/main.py]: "together"

File: akanda/rug/main.py [hunk 3]
```diff
@@ -29,7 +30,33 @@ def main(argv=sys.argv[1:]):
...
+        #             below, were definded in the manager module.
```
Comment (markmcclain) [inline akanda/rug/main.py]: defined

**Comment by markmcclain** [comment]
two minor items.. otherwise looks good for adding the purge.

**Pull request closed without merging**